nl/guides/new-course/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cursus aanmaken
+description: "Tutorial: cursus aanmaken"
+---
+
+# Cursus aanmaken
+
+Als lesgever kan je een nieuwe cursus aanmaken waar je studenten zich voor kunnen registreren. Dit doe je door op de knop `CURSUS AANMAKEN` te drukken in de rechterbovenhoek van het cursusoverzicht.
```

Comment:
Explain how a user gets to the "cursusoverzicht" page and link to it (https://dodona.ugent.be/nl/courses)